### ISSUE TITLE
PixelPilot_RK is broken after the latest updates

### ISSUE THREAD

**Issue by MarioFPVdev** [OpenIPC/PixelPilot_rk#35]
root@radxa-zero3:~/PixelPilot_rk# cmake -B build
-- The C compiler identification is GNU 10.2.1
-- The CXX compiler identification is GNU 10.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:30 (find_package):
  By not providing "Findspdlog.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "spdlog", but
  CMake did not find one.

  Could not find a package configuration file provided by "spdlog" with any
  of the following names:

    spdlogConfig.cmake
    spdlog-config.cmake

  Add the installation prefix of "spdlog" to CMAKE_PREFIX_PATH or set
  "spdlog_DIR" to a directory containing one of the above files.  If "spdlog"
  provides a separate development package or SDK, be sure it has been
  installed.

-- Configuring incomplete, errors occurred!
See also "/root/PixelPilot_rk/build/CMakeFiles/CMakeOutput.log".
root@radxa-zero3:~/PixelPilot_rk# sudo cmake --build build --target install
gmake: *** No rule to make target 'install'.  Stop.

**Comment by seriyps** [comment]
Please note the README update in this commit:
https://github.com/OpenIPC/PixelPilot_rk/commit/1ca5aabeb7ff6986d8c27f66740a20f156d52d1a#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R57

TLDR: you need to install `libspdlog-dev` DEB package

**Comment by MarioFPVdev** [comment]
Thanks. It worked after installing the package.
